fs/tarfs.py

```diff
@@ -225,10 +225,9 @@ def getinfo(self, path, namespaces=None):
                 raise errors.ResourceNotFound(path)
 
 
-            raw_tar_info = member.get_info(
-                *([self.encoding, None]
-                if six.PY2 else [])
-            )
+            raw_tar_info = member.get_info(*(
+                [self.encoding, None] if six.PY2 else []
```

Comment:
Yes, fonction signature is not the same between Py2 and Py3, so this makes it consistent.